<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>550</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Times New Roman</family>
   </font>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>База данных</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>621</width>
      <height>41</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QLabel" name="program_name_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>19</pointsize>
      </font>
     </property>
     <property name="text">
      <string>База данных обучающихся</string>
     </property>
    </widget>
    <widget class="QLabel" name="employee_name_label">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>0</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>16</pointsize>
       <weight>50</weight>
       <italic>true</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Имя
Фамилия</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="exit_btn">
     <property name="geometry">
      <rect>
       <x>555</x>
       <y>-5</y>
       <width>71</width>
       <height>53</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>15</pointsize>
       <underline>true</underline>
      </font>
     </property>
     <property name="text">
      <string>Выйти</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="search_edit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>521</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Введите имя или фамилию ребенка</string>
    </property>
   </widget>
   <widget class="QPushButton" name="search_btn">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>47</y>
      <width>75</width>
      <height>37</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>поиск</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_new_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>505</y>
      <width>113</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Новый</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="search_tbl">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>601</width>
      <height>411</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <property name="columnCount">
     <number>3</number>
    </property>
    <column>
     <property name="text">
      <string>Имя</string>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>17</pointsize>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Фамилия</string>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>17</pointsize>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Дата рождения</string>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>17</pointsize>
      </font>
     </property>
    </column>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
